internal/v1/api.go

```diff
@@ -801,6 +801,11 @@ type Readiness struct {
 	Readiness string `json:"readiness"`
 }
 
+// RecommendationsResponse defines model for RecommendationsResponse.
+type RecommendationsResponse struct {
+	Message *[]string `json:"message,omitempty"`
+}
```

Comment:
Why this is still named `Message`? From what I have hears, there are plans to break this response string into an array, I already see slice here so was this already done? Why this isn’t named `Packages` or something more relevant?